src/libs/metadata_view.c

```diff
@@ -141,6 +143,8 @@ static const char *_labels[] = {
   N_("exposure"),
   N_("exposure bias"),
   N_("focal length"),
+  N_("FF equiv focal length"),
```

Comment:
35mm instead of FF? (It is literally `FocalLengthIn35mmFilm` in the Exif spec; exiftool prints it as "Focal Length In 35mm Format".)

Reply:
Yes, 35mm would also be better for me.

Reply:
So "35mm equiv focal length"? I'm trying to keep it as short as possible

Reply:
done